include/simdjson/document.h

```diff
@@ -231,7 +238,11 @@ class object : protected internal::tape_ref {
    * Part of the std::iterable interface.
    */
   inline iterator end() const noexcept;
-
+  /**
+  * Get the size of the object (number of keys).
+  * It is a satured value with a maximum of 0xFFFFFF.
```

Comment:
satured -> saturated, but maybe better to avoid the word and say "maximum of 0xFFFFFF, if the value is 0xFFFFFF then there may be more than 0xFFFFFF entries." Just because saturated isn't a word people encounter often.

It's too early for this sort of comment, I'm sure :)